src/components/Public/Record/Importer/Progress.vue

```diff
@@ -13,13 +13,12 @@
                         class="progress-label"
                         variant="primary">
 
-          <span class="font-weight-bold">{{ $t('block.recordList.import.progress') }}</span>
```

Comment:
if this translation string is not used anywhere anymore, remove it from the i18n files.